tests/output/runner.rs

```diff
@@ -535,42 +534,43 @@ async fn test_testrun_step_error_with_details() -> Result<()> {
     .await
 }
 
-#[tokio::test]
-async fn test_testrun_step_scope_log() -> Result<()> {
-    let expected = [
-        json_schema_version(),
-        json_run_default_start(),
-        json_step_default_start(),
-        json!({
-            "sequenceNumber": 4,
-            "testStepArtifact": {
-                "log": {
-                    "message": "This is a log message with INFO severity",
-                    "severity": "INFO"
-                }
-            }
-        }),
-        json_step_complete(5),
-        json_run_pass(6),
-    ];
+// #[tokio::test]
```

Comment:
why commented and not removed?

Reply:
replied above; this is the test for the `scope` method, temporarily disabled